src/docs/docs_gen.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package docs
+
+//go:generate go run bash_completion.d/bash_completion.go "${BUILDDIR}/etc/bash_completion.d/singularity"
```

Comment:
I don't think this file is strictly necessary. Would be much simpler to just directly call go run from the makefile itself, see the example for the `singularity.conf` file